src/query/list_file.c

```diff
@@ -6315,8 +6313,7 @@ qfile_add_tuple_get_pos_in_list (THREAD_ENTRY * thread_p, QFILE_LIST_ID * list_i
       tuple_pos->vpid = list_id_p->last_vpid;
     }
 
-  offset = ((tuple_length + list_id_p->last_offset) > DB_PAGESIZE ? DB_PAGESIZE : tuple_length);
-  qfile_add_tuple_to_list_id (list_id_p, page_p, tuple_length, offset);
+  qfile_add_tuple_to_list_id (list_id_p, page_p, tuple_length, MIN (tuple_length, qfile_Max_tuple_page_size));
```

Comment:
MIN (tuple_length, qfile_Max_tuple_page_size) can be replaced with tuple_page_size